lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -193,15 +194,18 @@ export default function ContentSelector({
             variant="primary"
             data-testid="url-button"
           >
-            Enter URL of web page or PDF
+            <FormattedMessage
+              defaultMessage="Enter URL of web page or PDF"
+              description="button label: content selection"
+            />
           </LabeledButton>
           {canvasFilesEnabled && (
             <LabeledButton
               onClick={() => selectDialog('canvasFile')}
               variant="primary"
               data-testid="canvas-file-button"
             >
-              Select PDF from Canvas
+              <FormattedMessage defaultMessage="Select PDF from Canvas" />
```

Comment:
This example doesn't have a description; it's not required. Though I'd imagine we'd want one, probably. The audience for `description` is translators or anyone else who will be poring through the eventual message JSON.